src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -62,7 +62,7 @@ public void testPeekaboo() {
 	@Test
 	public void testCountOut() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;

```

Comment:
The original constant should be used (it would be best to use a constant from stacks interface).